File: Apps/Demos/RadialMenu/RadialMenuNet/MainWindow.cs
using SpaceVIL;
using SpaceVIL.Core;

namespace RadialMenu
{
    public class MainWindow : ActiveWindow
    {
        public ContactMenu Menu;
        public ButtonCore ShowContactsBtn;
        public RadialMenuItem RadialMenu;

        public override void InitWindow()
        {
            SetParameters("Mainwindow", "RadialMenu Example C#", 800, 600, false);
            SetMinSize(300, 300);
            SetBackground(StyleFactory.CommonBackground);
            SetPadding(0, 0, 0, 0);
            SetAntiAliasingQuality(MSAA.MSAA8x);
            IsCentered = true;

            // title
            TitleBar title = new TitleBar(GetWindowTitle());

            // radial menu 
            RadialMenu = new RadialMenuItem(this);

            // contact menu
            Menu = new ContactMenu(this, RadialMenu);

            ShowContactsBtn = new ButtonCore("Show");
            ShowContactsBtn.SetStyle(StyleFactory.GetRoundedButtonStyle());

            // adding
            AddItems(title, ShowContactsBtn);
        }
    }
}

File: Apps/Demos/RadialMenu/RadialMenuJVM/src/main/java/RadialMenu/MainWindow.java
package RadialMenu;

import com.spvessel.spacevil.ActiveWindow;
import com.spvessel.spacevil.ButtonCore;
import com.spvessel.spacevil.TitleBar;
import com.spvessel.spacevil.Flags.MSAA;

public class MainWindow extends ActiveWindow {

    ContactMenu menu;
    ButtonCore showContactsBtn;
    RadialMenuItem radialMenu;

    @Override
    public void initWindow() {
        setParameters("Mainwindow", "RadialMenu Example Java", 800, 600, false);
        setMinSize(300, 300);
        setBackground(StyleFactory.commonBackground);
        setPadding(0, 0, 0, 0);
        setAntiAliasingQuality(MSAA.MSAA_8X);
        isCentered = true;

        // title
        TitleBar title = new TitleBar(getWindowTitle());

        // radial menu
        radialMenu = new RadialMenuItem(this);

        // contact menu
        menu = new ContactMenu(this, radialMenu);
        
        // show contacts button
        showContactsBtn = new ButtonCore("Show");
        showContactsBtn.setStyle(StyleFactory.getRoundedButtonStyle());

        // adding
        addItems(title, showContactsBtn);
    }
}